src/main/java/org/minions/devfund/damianvp/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,150 @@
+package org.minions.devfund.damianvp.moviedatabase;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+
+/**
+ * MovieDatabase class.
+ */
+public class MovieDatabase {
+    private ArrayList<Movie> movieList;
+    private ArrayList<Actor> actorList;
+
+    /**
+     * Constructor class.
+     */
+    MovieDatabase() {
+        this.movieList = new ArrayList<>();
+        this.actorList = new ArrayList<>();
+    }
+
+    /**
+     * Method to get the movie list.
+     * @return ArrayList<Movie>, movieList.
+     */
+    ArrayList<Movie> getMovieList() {
+        return this.movieList;
+    }
+
+    /**
+     * Method to get the actor list.
+     * @return ArrayList<Actor>, actorList.
+     */
+    ArrayList<Actor> getActorList() {
+        return this.actorList;
+    }
+
+    /**
+     * Method to add a Movie.
+     * @param name String type.
+     * @param actors String List type
+     */
+    void addMovie(final String name, final String[] actors) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)